saleor/graphql/discount/mutations/promotion/promotion_create.py

```diff
@@ -200,6 +203,27 @@ def _save_m2m(
             rule.channels.set(channels)
 
     @classmethod
-    def send_sale_notifications(cls, manager, instance):
-        # TODO: implement the notifications
-        pass
+    def send_promotion_notifications(
+        cls, manager: "PluginsManager", instance: models.Promotion
+    ):
+        cls.call_event(manager.promotion_created, instance)
+        cls.send_promotion_toggle_notification(manager, instance)
+
+    @staticmethod
+    def send_promotion_toggle_notification(
+        manager: "PluginsManager", instance: models.Promotion
+    ):
+        """Send a notify about starting or ending promotion if it hasn't been sent yet.
```

Comment:
[Question] Is the description/naming ok? I mean for me `Send the notification` sounds like sending notification to the customers, but as a Saleor, we don't send any notification directly, we send the webhook events, and sending notification to customers can be done by external Saleor App.

Reply:
I changed the names